src/main/java/com/qa/hubspot/page/DealsPage.java

```diff
@@ -0,0 +1,11 @@
+package com.qa.hubspot.page;
+
+public class DealsPage {
+	
```

Comment:
where is the code?
where is the page locators
where is page actions

please complete this class